### PR TITLE
Landing page content and style

### PR DESCRIPTION
**Pull request by Drewbi**

### CODE REVIEW

File: client/src/layout/Header/index.js [hunk 1]
```diff
@@ -30,7 +30,9 @@ const Header = () => {
   return (
     <AppBar position="static" className={styles.header}>
       <Toolbar>
-        <Logo className={styles.logo}></Logo>
+        <Link to="/">
+          <Logo className={styles.logo}></Logo>
```
Comment (khattakdev) [inline client/src/layout/Header/index.js]:
```suggestion
          <Logo className={styles.logo}/>
```

File: client/src/pages/Landing/index.js [hunk 2]
```diff
@@ -1,12 +1,69 @@
...
+
+const renderSections = landingSections.map((section, i) => {
+  const Illustration = section.illustrationComponent;
   return (
-    <>
-      <h1>Landing</h1>
-    </>
+    <div className={styles.section} key={i}>
+      {i % 2 === 0 && (
+        <Illustration className={styles.illustration}></Illustration>
```
Comment (khattakdev) [inline client/src/pages/Landing/index.js]:
```suggestion
        <Illustration className={styles.illustration}/>
```

File: client/src/pages/Landing/index.js [hunk 3]
```diff
@@ -1,12 +1,69 @@
...
+      )}
+      <div className={styles.textblock}>
+        <h2 className={styles.textheader}>{section.header}</h2>
+        <p className={styles.textbody}>{section.body}</p>
+        {section.button && <LoginButton />}
+      </div>
+      {i % 2 !== 0 && (
+        <Illustration className={styles.illustration}></Illustration>
```
Comment (khattakdev) [inline client/src/pages/Landing/index.js]:
```suggestion
        <Illustration className={styles.illustration}/>
```

File: client/src/assets/Logo.svg [hunk 4]
```diff
@@ -1,32 +1,32 @@
...
         fill: #fff;
       }
 
-      .cls-2, .cls-3 {
+      .log-2, .log-3 {
         isolation: isolate;
       }
 
-      .cls-3 {
+      .log-3 {
```
Comment (khattakdev) [inline client/src/assets/Logo.svg:12]: Why do we have multiple CSS classes with the same values?

**Review by khattakdev — CHANGES_REQUESTED** [review]

File: client/src/assets/Logo.svg (hunk 4 above)
Comment (Drewbi) [reply client/src/assets/Logo.svg:12]: This is an auto generated svg, I changed the names to stop conflicts but no way am I optimising it by hand lmao

**Review by khattakdev — APPROVED** [review]

**Pull request merged**